pkg/converter/convert.go

```diff
@@ -149,7 +149,17 @@ func Convert(o Options, pathOptions *clientcmd.PathOptions) error {
 		return fmt.Errorf("unable to load kubeconfig: %s", err)
 	}
 
-	for _, authInfo := range config.AuthInfos {
+	targetAuthInfo := ""
+
+	if o.context != "" && config.Contexts[o.context] != nil {
+		targetAuthInfo = config.Contexts[o.context].AuthInfo
+	}
+
+	for name, authInfo := range config.AuthInfos {
+
+		if targetAuthInfo != "" && name != targetAuthInfo {
+			continue
```

Comment:
question: what's the expected behavior when there is no auth info matches the passed context name? Should we fail the call?

Reply:
it'd be no ops. meaning the kubeconfig will not be updated at all